docs/_Research-Data-Management/13-data-qc.md

```diff
@@ -1,196 +1,387 @@
+
+Possible reasons are: hard coded adapter sequences, .... 
+
+💡 adjust parameters  
+
+
+### 🚩 Wrongly Trimmed Reads 
+
+trimmed reads although no adapters are present. 
+
+Possible reasons are: false positive result of TG. 
+
+💡 skip trim-galore  
+
+
+### 🚩 Extremely low read counts (Mapping)
+
+After alignment almost all or a very large portion of the reads (>80%) are unmapped or the number of non-zero gene counts is very high.. 
+
+Possible reasons are: wrong target genome/transcriptome. 
+
+💡 -  
+
+
+### 🚩 Unmapped Reads (Mapping)
+
+high number of unmapped reads. 
+
+Possible reasons are: adapter sequence still present. 
+
+💡 go back to trimming  
+
+
+### 🚩 Correlation between biological conditions (Post-Mapping)
+
+different biological conditions correlate better than the replicates with each other. 
+
+Possible reasons are: low potential for DEGs present. 
+
+💡 There is no general solution.  
+
+
+### 🚩 Inconsistent Replicates (Post-Mapping)
+
+within replicate correlation is weak (r2<0.5). 
+
+Possible reasons are: e.g. contamination with HVG like rRNAs. 
+
+💡 if only a small sub-population is confounding -> remove  
+
+replicates do not cluster. 
+
+Possible reasons are: e.g. biol. and tech. replicates are mixed up. 
+
+💡 END-RESTART  
+
+replicates do not cluster. 
+
+Possible reasons are: negative control study. 
+
+
+
+
+### 🚩 Correlation between biological conditions (Post-Mapping)
+
+two experiment of 2 conditions (ctrl, KO) cluster in the wrong area. 
+
+Possible reasons are: human error, e.g. misslabeling of tubes. 
+
+💡 check design matrix, documentation, ...  
+
+no clear separation between conditions (ctrl, KO) . 
+
+Possible reasons are: e.g. biol. and tech. replicates are mixed up. 
+
+💡 END-RESTART  
+
+
+### 🚩 Dispersion Plot Anomalies (DEA)
+
+dispersion-plot: gene estimation does not follow red fit. 
+
+Possible reasons are: model does not represent data well. 
+
+💡 Deseq is not applicable  
+
+dispersion-plot: high fit dispersion for high mean count. 
+
+Possible reasons are: low number of replication + high variability. 
+
+💡 careful with reported DEGs  
+
+
+### 🚩 Predominantly Differentially Expressed Genes (DEA)
+
+almost all genes are DE (>90%). 
+
+Possible reasons are: e.g. biol. and tech. replicates are mixed up. 
+
+💡 END-RESTART  
+
+
+### 🚩 Sparse Differentially Expressed Genes (DEA)
+
+almost no gene is DE (<5%). 
+
+Possible reasons are: negative control study. 
+
+
+
+
+### 🚩 Wrong visualization metric (Visualize)
+
+using CPM to visualize DEGs. 
+
+Possible reasons are: mixed up within- and between-sample normalization. 
+
+💡 use correct normalization  
+
+use raw Ratios to visualize DEGs effect size. 
+
+Possible reasons are: humans are bad with ratios (0.01 = almost 0 and 100 is just large but not the largest bar ever). 
+
+💡 use any log transformation (e.g. log10: 0.01 => -2, 100 => +2)  
+
+
+# single cell
+
+
+### 🚩 Problem with per cell histogram (quality check)
```

Comment:
Is there any reason to use the level-3 headings instead of the level-2 `##` that would make browsing the sections more intuitive?

I could change the markdown but if this is generated, I'll leave it as a comment.